neo4j-5.26.yaml

```diff
@@ -38,24 +38,48 @@ pipeline:
 
       mvn package -DskipTests=true -T$(nproc)C -q -Doverwrite=true
 
-      mkdir -p ${{targets.destdir}}/usr/share/java/neo4j
-      tar --strip-components=1 -xf packaging/standalone/target/neo4j-*.tar.gz -C ${{targets.destdir}}/usr/share/java/neo4j
+      mkdir -p ${{targets.contextdir}}/var/lib/neo4j
+      # https://github.com/neo4j/docker-neo4j-publish/blob/b8c0ca2e8dd585e338d74b191a2e370c0da14d92/5.26.0/bullseye/community/Dockerfile#L20
+      tar --strip-components=1 -xf packaging/standalone/target/neo4j-*.tar.gz -C ${{targets.contextdir}}/var/lib/neo4j
 
-      mkdir -p ${{targets.destdir}}/usr/bin
+      mkdir -p ${{targets.contextdir}}/data
+      cp -r ${{targets.contextdir}}/var/lib/neo4j/data/* ${{targets.contextdir}}/data
+      ln -sf /var/lib/neo4j/data ${{targets.contextdir}}/data
+
+      mkdir -p ${{targets.contextdir}}/logs
+      mv ${{targets.contextdir}}/var/lib/neo4j/logs ${{targets.contextdir}}/
+      ln -sf /var/lib/neo4j/logs ${{targets.contextdir}}/logs
+
+      mkdir -p ${{targets.contextdir}}/usr/bin
       for i in neo4j neo4j-admin cypher-shell; do
-        ln -sf /usr/share/java/neo4j/bin/$i ${{targets.destdir}}/usr/bin/$i
+        ln -sf /var/lib/neo4j/bin/$i ${{targets.contextdir}}/usr/bin/$i
       done
 
 subpackages:
-  - name: ${{package.name}}-oci-entrypoint
-    description: OCI entrypoint for Neo4j
+  - name: ${{package.name}}-docker-publish
+    description: Docker image for Neo4j
     pipeline:
-      - runs: |
-          mkdir -p ${{targets.subpkgdir}}/usr/bin
-          mv entrypoint.sh ${{targets.subpkgdir}}/usr/bin/entrypoint.sh
+      - uses: git-checkout
+        with:
+          repository: https://github.com/neo4j/docker-neo4j-publish
```

Comment:
How are we going to check the updates of this github repo? Should we write a comment for future reviewers that we need another github to be pulled in so we need to manually check in the updates?

Reply:
this is definitely a fair concern! I'll take care of it!